defaults/main.yml

```diff
@@ -183,3 +183,6 @@ docker_remove_all: false
 # Additional files or directories to be remove if for example non-standard locations
 # was previously configured for data storage etc.
 docker_remove_additional: []
+# It bypasses GPG keys checking for package manager
+allow_unauthenticated: no
```

Comment:
Please change this to have docker_ prefix: docker_allow_unauthenticated

Update references to it accordingly.